src/components/Movies.js

```diff
@@ -25,22 +26,47 @@ function Movies({ filters, changeFilters }) {
     );
   }
 
+  const handleLikeClick = (e) => {
+    const movie_id = e.target.value;
+
+    reactOnMovie({ id: movie_id, reaction: 1 });
+  };
+
+  const handleDislikeClick = (e) => {
+    const movie_id = e.target.value;
+
+    reactOnMovie({ id: movie_id, reaction: 0 });
+  };
+
   return (
     <>
-      <div className="container">
-        <div className="characters">
-          {movies.data.map((movie) => {
-            return <Movie key={movie.id} movieData={movie} className="box" />;
-          })}
+      {movies && (
+        <div className="container">
+          <div className="characters">
+            {movies.data.map((movie) => {
+              const movieReaction = movie.reactions[0] ? movie.reactions[0] : 0;
```

Comment:
Mislim da bih izmenila response da ne vadimo korisnikovu reakciju iz nekog niza, nego da imamo samo jednu vrednost

Reply:
apsolutno se salzem ali nisam znao kako. Pitao sam i Nikolu ali mi on rece da ne komplikujem. Pa ako mozes voleo bih da mi kazes kako to da uradim. Mucio sam se sa elokventom i nisam uspeo. Jedino sto mi je palo na pamet je da modifikujem rezultat koji dobija preko elokventa iz baze?